blueflood-http/src/main/java/com/rackspacecloud/blueflood/http/DefaultHandler.java

```diff
@@ -16,14 +16,33 @@
 
 package com.rackspacecloud.blueflood.http;
 
+import com.codahale.metrics.Timer;
+import com.rackspacecloud.blueflood.io.Constants;
+import com.rackspacecloud.blueflood.utils.Metrics;
+import org.jboss.netty.buffer.ChannelBuffers;
 import org.jboss.netty.channel.ChannelHandlerContext;
-import org.jboss.netty.handler.codec.http.HttpRequest;
-import org.jboss.netty.handler.codec.http.HttpResponseStatus;
+import org.jboss.netty.handler.codec.http.*;
 
 public class DefaultHandler implements HttpRequestHandler {
+    private static final Timer sendResponseTimer = Metrics.timer(DefaultHandler.class, "HTTP Ingestion response sending timer");
 
     @Override
     public void handle(ChannelHandlerContext ctx, HttpRequest request) {
         HttpResponder.respond(ctx, request, HttpResponseStatus.OK);
     }
+
+    public static void sendResponse(ChannelHandlerContext channel, HttpRequest request, String messageBody, HttpResponseStatus status) {
```

Comment:
@chinmay-gupte i did not go through with that approach because the class HttpRollupHandler extends RollupHandler too and changing HttpRequestHandler to an abstract class would not have been possible.